packages/core/stories/card/card.stories.tsx

```diff
@@ -41,17 +50,24 @@ const Examples = () => (
   <>
     <ExampleRow name="Default">
       <Card>
-        <div>
-          <H1>Card with Density</H1>
-          <Text>Here is some content</Text>
-        </div>
+        <Text>Content</Text>
       </Card>
     </ExampleRow>
   </>
 );
 
+const InteractableExamples = () => (
+  <>
```

Comment:
Not a blocker - these fragments probably aren't needed (same for above Examples component)